src/component/mxgraph/MxClientConfigurator.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { mxgraph } from 'ts-mxgraph';
+
+declare const mxConstants: typeof mxgraph.mxConstants;
+declare const mxCodecRegistry: typeof mxgraph.mxCodecRegistry;
+declare const mxCodec: typeof mxgraph.mxCodec;
+
+export default class MxClientConfigurator {
+  public static configureMxCodec(): void {
```

Comment:
:question: we could explain why we add this
It comes from the ts-mxgraph (original repo) and it seems that it improves error handling when some mxGraph types are missing